### DISCUSSION TITLE
"Hide article from navigation" toggle

### DISCUSSION THREAD

**Discussion by sofietoft** [GitbookIO/community#12] [Feature Requests]
### What's your feature/idea?

Being able to hide an article from the navigation (public facing), but still include it in the search.

We use the Git Sync feature, and on some occasions, we would like to option to hide articles from the navigation, but still have them searchable.
Example: We have an article covering a temporary bug or solution. When this temporary issue/bug has been resolved, we would like to hide the article, but still keep it in the space, as people who haven't upgraded to the latest version, would still need the temporary solution.

### What's your desired outcome?

A toggle - possibly on the right-hand side when editing an article - that enabled you to **hide the article in the navigation**.

This could potentially also be a YAML tag to add in the markdown files, as this would be beneficial for the Git Sync feature.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

**Comment by aikoii** [comment]
@sofietoft You might already know, but you're now able to hide pages from the ToC 😊 👉 #675

**Reply by sofietoft** [reply]
We are aware, yes! But thanks for reaching out 🙌
